generators/client/templates/angular/src/main/webapp/app/shared/shared.module.ts.ejs

```diff
@@ -32,6 +32,7 @@ import { FormatMediumDatePipe } from './date/format-medium-date.pipe';
 import { SortByDirective } from './sort/sort-by.directive';
 import { SortDirective } from './sort/sort.directive';
 import { ItemCountComponent } from './pagination/item-count.component';
+import { FilterComponent } from './filter/filter.component';
```

Comment:
I don't think, I do have this variable available in the shared module folder as this is something entity specific. But sinc ealso the sorting directives are in there, I still thought it would be a good place to put it in.

Reply:
Hope it's fine that this would be then some general addition, so resolving this ;)